### PR TITLE
Modal + accordion tweak 

### LINKED ISSUE #60: Modal component
Screen Shot 2020-09-08 at 1 17 19 PM

https://share.goabstract.com/484b1932-f738-4a1e-b478-2af5ba9ab66d

### PR DESCRIPTION
**Pull request by jlhogan**
style tweak for accordions with headers with more information PLUS
closes #60

### CODE REVIEW

File: src/assets/images/index.tsx [hunk 1]
```diff
@@ -73,3 +74,6 @@ export const TeacherWithChalkboard = (props: React.SVGProps<SVGSVGElement>) => (
 export const TrashCan = (props: React.SVGProps<SVGSVGElement>) => (
   <SVGInline svgProps={props} url={trashCan} />
 );
+export const Cancel = (props: React.SVGProps<SVGSVGElement>) => (
```
Comment (mmazanec22) [inline src/assets/images/index.tsx:77]: Should the names of these maybe reflect what the image looks like rather than what we use it for?

**Review by mmazanec22 — APPROVED** [review]

File: src/assets/images/index.tsx (hunk 1 above)
Comment (jlhogan) [reply src/assets/images/index.tsx:77]: totally agreed -- `times` felt weird, but that is what font-awesome called it and i'm down to make that change if others agree it's clearer

Comment (mmazanec22) [reply src/assets/images/index.tsx:77]: xInCircle or something like that?

Comment (jlhogan) [reply src/assets/images/index.tsx]: renamed to `x` heh, since it's not in a circle. if we hate it we can always rename it

**Review by JordanGuinn — APPROVED** [review]

**Pull request merged**